packages/x-components/src/x-modules/facets/components/facets/facets.vue

```diff
@@ -105,6 +106,20 @@
     @Getter('facets', 'facets')
     public facets!: Record<Facet['id'], Facet>;
 
+    /**
+     * Transform a dictionary of Facets including the slot name.
+     *
+     * @returns A dictionary of facets with the slot name.
+     *
+     * @internal
+     */
+    protected get mappedFacets(): Dictionary<Facet> {
+      return map(this.facetsToRender, (_, facet) => ({
+        ...facet,
+        slotName: toKebabCase(facet.id as string)
```

Comment:
Follow the same pattern you did in other components, wrapping the facet into an object, and create an interface for the type. Facet type do not have `slotName`. It might make sense to extract this pattern to a utility type.